### ISSUE TITLE
How to handle typos

### ISSUE THREAD

**Issue by turion** [timokau/marvin-mk2#31]
When you mistype a `/status something` command and then edit the comment to correct it, this change is not picked up by marvin.

**Comment by timokau** [comment]
Yes, handling edits right is not easy. We'd want to avoid `/status something` in an old comment to be re-executed if you fix some unrelated typo. That would require parsing the diff, and even then it would probably no be easy to cover all the edge cases. Therefore I opted not to handle edits for now.

For what its worth, #29 will reduce the amount of explicit status commands you have to type a lot.
